<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>464</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(18, 18, 18)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
  background:  rgb(18, 18, 18);
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background:  rgb(29, 29, 29);
 }

QTabWidget::pane { 
   border: none;
}
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Language Detector</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>751</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="selectFile">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(100, 59, 182);
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
      <property name="text">
       <string>Select Audio File</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(62, 62, 62);
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 62, 62);
font-size: 10px;
border-radius: 3px;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="opText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>431</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(62, 62, 62);
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="run">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(100, 59, 182);
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="visual">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(100, 59, 182);
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
      <property name="text">
       <string>See Visualization</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="processText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
color: white;
border-radius: 3px;
font-size: 13px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="language_icon_leftside">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo.ico</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="by_runTimeterrorr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 12px;
color: white;
background: transparent;</string>
     </property>
     <property name="text">
      <string>by runTimeterrorr</string>
     </property>
    </widget>
    <widget class="QLabel" name="SIH_2020">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
     </property>
     <property name="text">
      <string>SIH 2020</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>About us</string>
    </attribute>
    <widget class="QGroupBox" name="rohan_card">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="rohan_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/rohan.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="rohan_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Rohan Sapkal</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="runTimeterrorr_logo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-40</y>
       <width>211</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/runtimeTerrorr_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="runtimeTerrorr_undeLogo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 25px;
color: white;
background: transparent;</string>
     </property>
     <property name="text">
      <string>runtimeTerrorr</string>
     </property>
    </widget>
    <widget class="QLabel" name="problem_info">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
     </property>
     <property name="text">
      <string>Problem Statement -
Open set Language Diarization for
Indian Languages audio data

Organization - 
Bharat Electronics Limited , Bengaluru

Category - 
Software</string>
     </property>
    </widget>
    <widget class="QLabel" name="dypiemr_add">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
     </property>
     <property name="text">
      <string>Dr. D.Y. Patil Institute of Engineering,
Management and Research
Pimpri Chinchwad, Maharashtra</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="roshan_card">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="roshan_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/roshan.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="roshan_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Roshan Pawar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="sumit_card">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>0</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="sumit_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/sumit.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="sumit_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Sumit Pawar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="saurabh_card">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="saurabh_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/saurabh.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="saurabh_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Saurabh Pawar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="vishal_card">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>220</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="vishal_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/vishal.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="vishal_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Vishal Jagtap</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="neha_card">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>220</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(29, 29, 29);
color: white;
border-radius: 5px;
font-size: 13px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="neha_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/neha.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="neha_name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: white;
background: transparent;</string>
      </property>
      <property name="text">
       <string>Neha More</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
